src/main/java/com/sourcegraph/website/OpenRevisionAction.java

```diff
@@ -22,8 +22,7 @@
 import org.jetbrains.annotations.NotNull;
 
 /** JetBrains IDE action to open a selected revision in Sourcegraph. */
-@SuppressWarnings("MissingActionUpdateThread")
-public class OpenRevisionAction extends DumbAwareAction {
+public class OpenRevisionAction extends DumbAwareBGTAction {
```

Comment:
Unfortunately there are more actions to fix lie this.

Reply:
Updated